src/js/edu-benefits/5490/config/form.js

```diff
@@ -91,15 +89,58 @@ const formConfig = {
       pages: {
         applicantInformation: applicantInformation(fullSchema5490, {
           labels: { relationship: relationshipLabels }
-        })
+        }),
+        additionalBenefits: additionalBenefits(fullSchema5490, {
+          fields: ['civilianBenefitsAssistance', 'civilianBenefitsSource']
+        }),
+        applicantService: {
+          title: 'Applicant service',
```

Comment:
The only clear answer I've gotten is that chapter titles should be title case and everything else should be sentence case.

For this particular line, that's a page title and you'll only see it on the review page.